src/output/index.js

```diff
@@ -231,10 +234,19 @@ export default class Output {
   }
 
   logError (err: Error | string) {
+    this._logToFile(err, this.errlog)
+  }
+
+  logAutocomplete (msg: string) {
```

Comment:
I don't think we need this, but I think it might just be an artifact from an earlier version of your code